docs/_articles/en/codebuilder/cb-diffs.md

```diff
@@ -0,0 +1,24 @@
+---
+title: What's Different
+lang: en
+---
+## What’s Different in Code Builder
+
+Code Builder offers the same rich tooling and unleashes the power of Salesforce VS Code extensions  as the desktop version. No matter whether you are using VS Code on desktop or Code Builder from a browser, the Salesforce extensions you access are the same. While the extensions generally function the same way regardless of how they are being accessed, there are some nuances to be aware of:
+
+* Code Builder installed as a managed package, and your Code Builder environment comes pre-configured for you.
+* Working in Code Builder doesn’t require you to install VS Code or Salesforce Extensions or Salesforce CLI. 
```

Comment:
I would simplify: Code Builder comes with everything you need: VS Code, Salesforce extensions, and Salesforce CLI. No installation required.

Reply:
Or --
With Code Builder, you're good to go. We install everything you need: VS Code, SF extensions, and Salesforce CLI.